osprey/processes/wps_convolution.py

```diff
@@ -0,0 +1,129 @@
+from pywps import Process, LiteralInput, ComplexOutput, FORMATS
+from pywps.app.Common import Metadata
+
+from rvic.convolution import convolution
+from rvic.core.config import read_config
+from rvic.version import version
+
+from datetime import datetime, timedelta
+
+import os
+import logging
+import json
+
```

Comment:
Could we use `wps_tools` utils and io like you did in `wps_parameters`?

Reply:
I'm gonna save that for a different PR since Nik already assigned me a separate issue for that.